jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -753,6 +754,16 @@ function configure_jbpm_cache() {
         </cache-container>\n<cache-container name="server" aliases="singleton cluster" default-cache="default" module="org.wildfly.clustering.server">#g' ${CONFIG_FILE}
 }
 
+function configure_jbpm_cache_modules() {
+    sed -i 's#<system export="true">#<module name="org.infinispan" services="export"/><module name="org.jgroups"/>\n<system export="true">#g' ${JBOSS_HOME}/standalone/deployments/ROOT.war/WEB-INF/jboss-deployment-structure.xml
+
+    if [ -d "/opt/kie/dependencies/jbpm-clustering" ] ;then
+      for f in /opt/kie/dependencies/jbpm-clustering/kie-server-services-jbpm-cluster-*.jar; do
+        mv "$f" ${JBOSS_HOME}/standalone/deployments/ROOT.war/WEB-INF/lib
```

Comment:
does symlink works? it so, please use symlink instead of moving the file.

Reply:
Symlink doesn't works with WEB-INF/lib jars

Reply:
ok, thanks.